src/Microsoft.Health.Fhir.SqlServer/Features/Exceptions/OperationFailedException.cs

```diff
@@ -0,0 +1,19 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.SqlServer.Features.Exceptions
+{
+    public class OperationFailedException : Exception
```

Comment:
We already have an `OperationFailedException` class in `Fhir.Core`. Can you rename this to something more specific to Sql to differentiate?